### ISSUE TITLE
Tests are missing

### ISSUE THREAD

**Issue by mochaaP** [Cu3PO42/gtk-session-lock#7]
Currently, the tests are straight from `gtk-layer-shell` and won't compile with `-Dtests=true`.

**Comment by Cu3PO42** [comment]
You are right. I thought I had fully disabled tests, but apparently hadn't.

I wanted to leave some of the intfrastructure in place to make it easier to add tests in the future, however.

I currently do not have the bandwidth/desire to work on tests for this library. I will gladly accept contributions in that direction, though.
